backstack/build.gradle.kts

```diff
@@ -14,6 +14,18 @@ kotlin {
   jvm()
   ios()
   iosSimulatorArm64()
+  js(IR) {
+    moduleName = property("POM_ARTIFACT_ID").toString()
+    compilations.all {
+      kotlinOptions {
+        sourceMap = true
+        moduleKind = "umd"
+        metaInfo = true
+      }
+    }
+    browser()
+    nodejs()
```

Comment:
You do not need both of these. Where do you want tests to run? Pick that one. I would choose nodejs–the library is not interacting with browser APIs that I can see.

Reply:
Gotcha, yeah I wasn't sure if these were like targets where we have to add each supported one or just picking a runtime

[bcc3fd6](https://github.com/slackhq/circuit/pull/801/commits/bcc3fd6954a4d0f5e1c1ea9b85c19bf048808786)